Gemfile

```diff
@@ -75,23 +75,22 @@ gem 'rest-client'
 gem 'noid', '~> 0.8'
 
 group :test do
-  gem "rspec-its"
+  gem "capybara"
+  gem "capybara-select2"
+  gem "database_cleaner"
   gem "ci_reporter_rspec"
+  gem "poltergeist", "~> 1.5"
+  gem "show_me_the_cookies"
   gem "timecop"
 end
 
 group :development, :test do
   gem "byebug"
   gem "rspec-rails"
   gem "ruby-debug-passenger"
-  gem "selenium-webdriver"
+  gem "selenium-webdriver" # used spec-views (legacy?)
   gem "jettywrapper"
-  gem "capybara"
-  gem "poltergeist", "~> 1.5"
   gem "factory_girl_rails"
-  gem "database_cleaner"
-  gem "capybara-select2"
-  gem "show_me_the_cookies"
```

Comment:
Can't see any reason why most of these test gems are in the development group as well? Some of them are valid (factory_girl/rspec-rails has generators) but these other ones I moved up to the test group.